CI-111 unit testing

wilrnh opened the pull request:
This PR just runs the unit tests on CircleCI and spit out any warnings/errors into the PR comments.

cameo-github commented:
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><strong><a href="https://github.com/vimeo/VimeoNetworking/blob/476a31148ded451d82079f465b63c474a15c31fb/VimeoNetworking/Sources/VimeoClient.swift#L316">VimeoNetworking/Sources/VimeoClient.swift#L316</a></strong>: cast from ‘Response&lt;VIMNullResponse&gt;’ to unrelated type ‘Response&lt;ModelType&gt;’ always fails  <br /><code>
                let response = Response(model: nullResponseObject, json: [:]) as! Response&lt;ModelType&gt;
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 20 tests, with 0 failures (0 unexpected) in 0.275 (0.290) seconds</td>
    </tr>
  </tbody>
</table>

See build details on [CircleCI](https://circleci.com/gh/vimeo/VimeoNetworking/83)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

nicolelehrer commented on Podfile.lock:
```diff
@@ -23,12 +23,12 @@ DEPENDENCIES:
 PODFILE CHECKSUM: 9f9867bb3720cc28bbfeab258d411b2ca6ee7bef
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.0.1
```
@wilrnh +1 with @jasonhawkins comment

nicolelehrer commented:
@wilrnh Unit tests pass - once you update cocoapods, resolve merge conflict, let us know if you need another review, otherwise you're good to go.

wilrnh commented:
@jasonhawkins @nicolelehrer thx for taking a look! I updated to cocoa pods 1.1.1 (seems like this one was never bumped from 1.0.1) and rebased with develop. If it looks good I just need an approve now thxxxxxx.

wilrnh commented:
🙏
